### PR TITLE
Update .appdata and .desktop files

### PR DESCRIPTION
**Pull request by jnumm**
To keep up with the following changes:
/usr/share/appdata has been changed to /usr/share/metainfo.
AppStream spec and desktop entry spec now recommend reverse-domain file names.
Appstream XML spec has been changed and extended.

https://www.freedesktop.org/software/appstream/docs/
https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html

### CODE REVIEW

**Comment by Wuzzy2** [comment]
Thank you! Usually files like these catch dust really fast in free software projects.

**Comment by est31** [comment]
@paramat @nerzhul @sofar can you have a look?

File: misc/net.minetest.minetest.appdata.xml [hunk 1]
```diff
@@ -42,5 +48,11 @@
...
 	<url type="donation">http://www.minetest.net/development/#donate</url>
-	<updatecontact>sfan5@live.de</updatecontact>
-</application>
+	<url type="faq">http://wiki.minetest.net/FAQ</url>
+	<url type="help">http://wiki.minetest.net</url>
+	<provides>
+		<binary>minetest</binary>
+	</provides>
+	<translation type="gettext">minetest</translation>
+	<update_contact>sfan5@live.de</update_contact>
```
Comment (rubenwardy) [inline misc/net.minetest.minetest.appdata.xml]: why sfan5? Why not a generic @minetest.net email

Comment (sfan5) [inline misc/net.minetest.minetest.appdata.xml]: If you look at the diff carefully you can see that this didn't actually change.

When an `appdata` file was added an update contact was needed and nobody wanted to do it, so my email was put there. Also we can't do a "generic" email on @minetest.net afaik.

**Pull request merged**